packages/grafana-data/src/transformations/transformers/groupingToMatrix.test.ts

```diff
@@ -105,6 +105,43 @@ describe('Grouping to Matrix', () => {
     });
   });
 
+  it('generates Matrix with empty entries', async () => {
+    const cfg: DataTransformerConfig<GroupingToMatrixTransformerOptions> = {
+      id: DataTransformerID.groupingToMatrix,
+      options: {
+        emptyValue: SpecialValue.Null,
+      },
+    };
+
+    const seriesA = toDataFrame({
+      name: 'A',
+      fields: [
+        { name: 'Time', type: FieldType.time, values: [1000] },
+        { name: 'Value', type: FieldType.number, values: [1] },
+      ],
+    });
+
+    await expect(transformDataFrame([cfg], [seriesA])).toEmitValuesWith((received) => {
+      const processed = received[0];
+      const expected: Field[] = [
+        {
+          name: 'Time\\Time',
+          type: FieldType.string,
+          values: new ArrayVector([1000]),
+          config: {},
+        },
+        {
+          name: '1000',
+          type: FieldType.number,
+          values: new ArrayVector([1, null]),
```

Comment:
@kylebrandt This is an example of "No data" right? https://github.com/grafana/grafana-plugin-sdk-go/tree/main/data/contract_docs#no-data-and-empty